src/qw/local_store/main.py

```diff
@@ -135,3 +136,23 @@ def write_templates_and_ci(self, service: GitService, *, force: bool):
                 )
             else:
                 shutil.copy(source_path, target_path)
+
+    def release_word_templates(self, out_dir=None):
+        """
+        Iterate through qw_release_templates/*.docx files.
+
+        Returns a pair (in_path, out_path) where in_path is the path
+        of the template file and out_path is the path it should be
+        written to.
+        """
+        if out_dir is None:
+            out_dir = self.base_dir / "qw_release_out"
+        top = self.base_dir / "qw_release_templates"
+        for dirpath, _dirnames, filenames in os.walk(top):
+            out_path = out_dir / os.path.relpath(dirpath, top)
```

Comment:
FYI I think you can use pathlib's `Path.relative_to` here so you don't have to use the two ways of doing it mixed together. Oh though `Path.walk` is only python 3.12 which is sad